RocketChat/Rocket.Chat.ReactNative
# [NEW] Flow to resend email confirmation from mobile

## Proposed changes

When a non-verified account tries to log in, the app gives out a "credentials were rejected" error message. Potentially misleading the user. The web expects this case and gives the option to send another "confirmation email" so the user is able to do it before trying to log in again.

So, we used the same idea to create a new flow to move the non-verified user to the screen to resend the email confirmation

## Issue(s)	


## How to test or reproduce

- create a new account on `open.rocket.chat`
- without verifying it, try to log in. 

## Screenshots

### Before

https://user-images.githubusercontent.com/47038980/137344684-93ff0a4f-0f7a-4822-a857-2f78aacf5e7c.mp4


### After

https://user-images.githubusercontent.com/47038980/137343691-35514262-02b1-4c74-93e4-98e89ad8c6e6.mp4




## Types of changes




## Checklist



## Further comments
